src/jit/codegenxarch.cpp

```diff
@@ -7947,7 +7947,7 @@ CodeGen::getFirstArgWithStackSlot()
 #else
```

Comment:
While you're here, the "#elif _TARGET_AMD64_" needs to be "#elif defined(_TARGET_AMD64_)